prairielib/lib/migrations.js

```diff
@@ -34,136 +35,272 @@ module.exports.init = function (dir, proj, callback) {
   });
 };
 
+/**
+ * Timestamp prefixes will be of the form `YYYYMMDDHHMMSS`, which will have 14 digits.
```

Comment:
Do we care about the timezone? I think not really, but maybe we should define these as UTC just for completeness? And could include a comment about how to make these timestamp strings? The simplest expression I could find is:
```
node -e "console.log(new Date().toISOString().replace(/\D/g,'').slice(0,14))"
```
Maybe we should add a `README.md` to the `migrations/` directory with information about the filename format and instructions on how to generate a new name?

Reply:
I like that idea, yeah. Maybe even a Makefile target like `make new-migration` that'll run that bit of Node code? I'll write some docs and think about a Makefile change.

Reply:
I added a README in 53b4703.